### PR TITLE
KeyboardAccessoryView - adding 'usesBottomTabs' prop

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
KeyboardAccessoryView - adding 'usesBottomTabs' prop to add bottom insets.

## Changelog
KeyboardAccessoryView - adding 'usesBottomTabs' prop to add bottom insets.

### CODE REVIEW

File: lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView/index.tsx [hunk 1]
```diff
@@ -20,6 +20,10 @@ export type KeyboardTrackingViewProps = ViewProps & {
     * Allow control safe area
     */
    useSafeArea?: boolean;
+   /**
+    * Whether or not to include bottom tab bat inset
```
Comment (ethanshar) [inline lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView/index.tsx:24]:
```suggestion
    * Whether or not to include bottom tab bar inset
```

**Review by ethanshar — APPROVED** [review]
See small type suggestion, Approved anyway. 
Let me know after it's merged, ill publish the version

**Comment by Inbal-Tish** [comment]
> See small type suggestion, Approved anyway. Let me know after it's merged, ill publish the version

Merged. Please publish. Thanks

**Comment by ethanshar** [comment]
> > See small type suggestion, Approved anyway. Let me know after it's merged, ill publish the version
> 
> Merged. Please publish. Thanks

Published 👌

**Pull request merged**